src/apps/material/helper.ts

```diff
@@ -245,3 +246,8 @@ export const getBestMaterialTypeForWork = (work: Work) => {
   return getManifestationsWithMaterialType(work.manifestations.all)[0]
     .materialTypes[0].specific;
 };
+
+export const reservationModalId = (faustIds: FaustId[]) => {
+  const sortedFaustIds = faustIds.sort();
+  return `reservation-modal-${sortedFaustIds.join("-")}`;
```

Comment:
Really a minor. But since `sortedFaustIds` is only used once, and no other checks are used, you might as well write:
```typescript
return `reservation-modal-${faustIds.sort().join("-")}`;
```
A teeny weeny memory optimisation.